CMakeLists.txt

```diff
@@ -36,6 +36,9 @@ elseif (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
 	add_definitions(-DRW_OSX)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
 	add_definitions(-DRW_FREEBSD)
+elseif(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
```

Comment:
nit: Commit name "OpenBSD support" or something like that would have been better (to suggest that OpenBSD wasn't supported previously). "Fixes" usually implies something was existing + broken.